modules/installation-launching-installer.adoc

```diff
@@ -90,10 +94,6 @@ ifeval::["{context}" == "installing-aws-default"]
 :no-config:
 :aws:
 endif::[]
-ifeval::["{context}" == "installing-aws-localzone"]
```

Comment:
Removing this `ifeval` will break the AWS Local Zones doc that shipped with 4.12.

Reply:
@mjpytlak this file has been replaced with new one and no longer has that problem.